inst/private/python_header.py

```diff
@@ -11,16 +11,12 @@
 sys.ps1 = ""; sys.ps2 = ""
 
 
-def myerr(e):
-    # hardcoded in case no xml
-    print("<output_block>")
-    print("<item>\n<f>9999</f>\n<f>")
-    print(str(e[0]).replace("&", "&amp;").replace("<", "&lt;").replace(">", "&gt;"))
-    print("</f><f>")
-    print(str(e[1]).replace("&", "&amp;").replace("<", "&lt;").replace(">", "&gt;"))
-    print("</f>\n</item>")
-    print("</output_block>\n")
-
+def myerr(error):
+    try:
+        octoutput_drv(("INTERNAL_PYTHON_ERROR", error))
+    except:
+        pass
+    raise
```

Comment:
aa, this solve a little bug, basically when we have an error here before two things, first the system was stalled waiting some return, and second, the error message was passed as ans, myerr now handle the python error and say to python_cmd was an error there to avoid stall it, and then show the error with raise and stop the script, the try its only for when fails ```octoutput_drv``` or when not is defined following your advice.
wen i test the works fine, so i suppose its safe do this, at least now we don't "leave a mess"